Cargo.lock

```diff
@@ -1,1434 +1,1591 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```

Comment:
please do it this way:
cargo +1.40.0 update

Reply:
Ah, thanks for that. I wasn't sure that `Cargo.lock` even belonged here, but the tests were failing without it.